service-api/module/Application/test/TestData/test-private-key.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Private keys can trigger various security tooling looking for leaked secrets (even if they are obviously test ones). Might be worth making a make target that generates these instead of committing, just saves time with queries about false positives later.